src/robusta/integrations/kubernetes/base_triggers.py

```diff
@@ -40,6 +40,22 @@ class K8sBaseTrigger(BaseTrigger):
     operation: K8sOperationType = None
     name_prefix: str = None
     namespace_prefix: str = None
+    labels_selector: str = None
+    _labels_map: dict = PrivateAttr()
```

Comment:
I think this has to be `Dict` (from `typing`) and not `dict` for compatibility with lower python versions

Reply:
Fixing